sdks/go/test/build.gradle

```diff
@@ -166,3 +174,33 @@ task ulrValidatesRunner {
     }
   }
 }
+
+// A method for configuring a cross-language validates runner test task,
+// intended to be used in calls to createCrossLanguageValidatesRunnerTask.
+ext.goIoValidatesRunnerTask = { proj, name, scriptOpts ->
+  return proj.tasks.create(name: name) {
+    group = "Verification"
+
+    dependsOn ":sdks:go:test:goBuild"
+    dependsOn ":sdks:go:container:docker"
+    dependsOn ":sdks:java:container:java11:docker"
+    dependsOn ":sdks:java:testing:expansion-service:buildTestExpansionServiceJar"
+    dependsOn ":sdks:java:io:expansion-service:build"
```

Comment:
Does `:build` also create `jar`? Do we need `:shadowJar`?

Reply:
I was pretty sure it required `shadowJar`, but I tried `jar` again just to be sure. I got this error in my command line shortly after starting the expansion service jar, followed by it crashing: `no main manifest attribute, in [...]/beam-sdks-java-io-expansion-service-2.33.0-SNAPSHOT-unshaded.jar`

This might be fixable, but I'll just stick with a shaded jar for now.